generate.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+#
+# This script generates the API implementations based off the current production API schemas.
+#
+set -eo pipefail
+shopt -u nullglob
+
+if [ -z "$(which docker)" ]; then
+	echo "Error: Docker must be available in order to run this script"
+	exit 1
+fi
+
+OPEN_CLONE_DIR=$(mktemp -d)
+OPEN_REPO="https://github.com/mitodl/mit-open.git"
+
+GENERATOR_VERSION=v7.2.0
+GENERATOR_IMAGE=openapitools/openapi-generator-cli:${GENERATOR_VERSION}
+
+pushd $OPEN_CLONE_DIR
+
+git clone --filter=blob:none $OPEN_REPO $OPEN_CLONE_DIR
+git checkout release
+
+popd
+
+
+for cfile in ./config/*.yaml ; do
+  echo "Generating: $cfile"
+  docker run --rm \
+    -v "${PWD}:/local" \
+    -v "${OPEN_CLONE_DIR}:/open" \
+    $GENERATOR_IMAGE \
+    generate \
+    --ignore-file-override "/local/.openapi-generator-ignore" \
+    -c "/local/$cfile"
+done
+
+
+# set permissions 
+docker run --rm \
+    -v "${PWD}:/local" \
+    --entrypoint /bin/bash \
+    $GENERATOR_IMAGE \
+    chown --reference /local -R /local/src/
```

Comment:
When I run `./generate.sh`, the client generates successfully but I also get an error:

```
################################################################################
# Thanks for using OpenAPI Generator.                                          #
# Please consider donation to help us maintain this project 🙏                 #
# https://opencollective.com/openapi_generator/donate                          #
################################################################################
/usr/bin/chown: /usr/bin/chown: cannot execute binary file
```

I think from this line?

Reply:
yeah, I had it working at some point so I'm not sure what regressed it.